resources/user-agents/browsers/safari/safari-10-0.json

```diff
@@ -1,7 +1,7 @@
 {
   "division": "Safari #MAJORVER#.#MINORVER#",
-  "versions": ["9.1", 9],
```

Comment:
I had to break up these files a little bit for the CFNetwork patterns so they could live in a versioned file that only contains one version.